portfolio/src/main/webapp/style.css

```diff
@@ -135,3 +176,8 @@ main > .comments-section .comments-form > :not(:last-child) {
 .space-between-flex > :first-child {
   margin-right: var(--spacing-md);
 }
+
+.centerInFlex {
```

Comment:
use kebab case for css selectors `.center-in-flex`

Reply:
Done in latest changes